Multisig/UI/Transaction/PaymentMethod/ChoosePaymentViewController.swift

```diff
@@ -93,15 +109,30 @@ class ChoosePaymentViewController: UIViewController, UITableViewDelegate, UITabl
 
     private func buildSignerAccountCell(tableView: UITableView) -> UITableViewCell {
         let cell = tableView.dequeueCell(PaymentMethodCell.self)
+        cell.selectionStyle = .none
         cell.accessoryType = .none
         cell.setSignerAccount()
         return cell
     }
 
     private func buildRelayerCell(tableView: UITableView) -> UITableViewCell {
         let cell = tableView.dequeueCell(PaymentMethodCell.self)
+        cell.selectionStyle = .none
         cell.accessoryType = .none
         cell.setRelaying(relaysRemaining, relaysLimit)
         return cell
     }
+
+    private func selectSignerAccountOption() {
+        tableView.selectRow(at: IndexPath(row: 2, section: 0), animated: false, scrollPosition: .none)
```

Comment:
I just don't like this magic constants here and below, it will make it harder to scale in future, can you please define constants or enum to encapsulate them ?